mongo/integration/unified/unified_spec_runner.go

```diff
@@ -26,6 +26,39 @@ var (
 		// GODRIVER-1773: This test runs a "find" with limit=4 and batchSize=3. It expects batchSize values of three for
 		// the "find" and one for the "getMore", but we send three for both.
 		"A successful find event with a getmore and the server kills the cursor (<= 4.4)": {},
+		// TODO(CSOT): Unskip these CSOT spec tests once createChangeStream operations are correctly formatted.
```

Comment:
See my [comment](https://github.com/mongodb/specifications/pull/960#discussion_r861070540) in the spec PR:
> The Go driver expects a `saveResultAsEntity` field for all the `createChangeStream` operations in these new tests and errors in its absence. The unified test format does not seem to imply that there _must_ be a `saveResultAsEntity` field for `createChangeStream`, so I _could_ just throw away the result of the `watch` command, but wanted to point out that this deviates from existing `createChangeStream` operations (by nature of the template used to create the tests).